crates/networking/rpc/eth/account.rs

```diff
@@ -159,15 +160,32 @@ impl RpcHandler for GetTransactionCountRequest {
             self.address, self.block
         );
 
-        let Some(block_number) = self.block.resolve_block_number(&context.storage)? else {
-            return serde_json::to_value("0x0")
-                .map_err(|error| RpcErr::Internal(error.to_string()));
+        let has_pending_tag = matches!(
```

Comment:
I think you can do something like https://github.com/lambdaclass/lambda_ethereum_rust/blob/134bc9469ac5e5eda46c6c287e5a3d970044e766/crates/networking/rpc/types/block_identifier.rs#L109 because of https://github.com/lambdaclass/lambda_ethereum_rust/blob/134bc9469ac5e5eda46c6c287e5a3d970044e766/crates/networking/rpc/types/block_identifier.rs#L152-L159

Reply:
I'll check it rn, great catch!